Sources/Classes/PDFViewController.swift

```diff
@@ -84,8 +84,17 @@ public final class PDFViewController: UIViewController {
     /// UIBarButtonItem used to override the default action button
     fileprivate var actionButton: UIBarButtonItem?
     
+    /// Default background color applied both to the view and the collectionView.
+    fileprivate var defaultBackgroundColor: UIColor {
```

Comment:
Using Swift3 naming conventions, we should remove the ```BackgroundColor``` segment of this name to just leave ```default```.  This will match what Apple have done in the standard library, for example they have ```.red``` and ```.white```.